package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -909,6 +924,7 @@ static const struct ubus_method bss_methods[] = {
 	UBUS_METHOD_NOARG("wps_start", hostapd_bss_wps_start),
 	UBUS_METHOD_NOARG("wps_cancel", hostapd_bss_wps_cancel),
 	UBUS_METHOD_NOARG("update_beacon", hostapd_bss_update_beacon),
+	UBUS_METHOD_NOARG("get_infos", hostapd_bss_get_infos),
```

Comment:
would "features" or "caps" not be a better name to use ?

Reply:
I thought infos is more general and maybe someone else want to extend this call by different information elements.
I don't mind. I can rename it if you want.